src/Mvc/test/WebSites/FilesWebSite/Controllers/DownloadFilesController.cs

```diff
@@ -47,7 +49,12 @@ public IActionResult DownloadFromDiskWithFileName_WithLastModifiedAndEtag()
     public IActionResult DownloadFromDiskSymlink()
     {
         var path = Path.Combine(_hostingEnvironment.ContentRootPath, "sample.txt");
-        var symlink = Path.Combine(Path.GetTempPath(), Path.GetRandomFileName());
+        var symlink = Path.Combine(_testFilesPath, Path.GetRandomFileName());
+
+        if (!Directory.Exists(_testFilesPath))
+        {
+            Directory.CreateDirectory(_testFilesPath);
```

Comment:
Super nit, but Directory.CreateDirectory no-ops if the directory already exists (so you can safely call it without the exists check)